src/test/java/pages/MyAccountPage.java

```diff
@@ -0,0 +1,144 @@
+package pages;
+
+import elements.MyAccountElements;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+
+public class MyAccountPage extends MyAccountElements {
+
+    public MyAccountPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public MyAccountPage clickAccountLogoutBtn() {
+        getAccountLogoutBtn().click();
+        return this;
+    }
+
+    public MyAccountPage clickAddNewAddress() {
+        getAddNewAddress().click();
+        return this;
+    }
+
+    public boolean isVisibleAddNewAddress() {
+        return getAddNewAddress().isDisplayed();
+    }
+
+    public MyAccountPage fillAddressTitleField() {
+        getAddressTitleField().sendKeys("123");
+        return this;
+    }
+
+    public MyAccountPage clickCountry() {
+        getCountrySelect().click();
+        return this;
+    }
+
+    public MyAccountPage fillFirstNameField() {
+        getFirstNameField().sendKeys("1234");
+        return this;
```

Comment:
Maybe change signature of your method. You can write string parameter and pass the parameter in the test...